src/core/projectTeams/test/projectTeam.service.spec.ts

```
@@ -1,1024 +1,1335 @@
+     *    - Send ProjectTeamDetailResponse to IndexService
+     * 11. Return ProjectTeamDetailResponse
+     */
+
+    // 1. Calculate total recruit member count
+    // - If count == 0, set isRecruit = false
+    beforeEach(async () => {
+        jest.spyOn(awsService, 'imageUploadToS3').mockResolvedValue(
+            'image.jpg',
+        );
+        jest.spyOn(indexService, 'createIndex').mockResolvedValue(undefined);
+    });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)